src/OpenTelemetry.Api/Trace/SpanHelper.cs

```diff
@@ -141,7 +139,7 @@ public static Status ResolveCanonicalCodeToStatus(string statusCanonicalCode)
 
             if (!success)
             {
-                return DefaultStatus;
+                return default;
```

Comment:
@rajkumar-rangaraj mentioned once that returning default is not as faster as returning the static DefaultStatus.. not sure if its relevant, as Status is struct...

Reply:
i will try some things and pst here the benchmark! :)

Reply:
i rollback my changes. the default variable is a little better.